config/v201/component_config/standardized/SecurityCtrlr.json

```diff
@@ -177,6 +177,38 @@
       "maximum": 3,
       "default": "1",
       "type": "integer"
+    },
+    "AllowCSMSRootCertificateInstallWhenLowSecurityProfile": {
+      "variable_name": "AllowCSMSRootCertificateInstallWhenLowSecurityProfile",
+      "characteristics": {
+        "supportsMonitoring": true,
+        "dataType": "boolean"
+      },
+      "attributes": [
+        {
+          "type": "Actual",
+          "mutability": "ReadWrite"
```

Comment:
What is the reason you've chosen ReadWrite here? Keep in mind the reason for this change is to prevent parties to install wrong certificates.

Reply:
I thought `ReadOnly` is the same as unmutable. now it is `ReadOnly`